modMcf/build.xml

```diff
@@ -141,6 +141,7 @@
 				<pathelement location="${dir.lib}/libnsc.jar" />
 				<pathelement location="${dir.lib}/commons-collections4-4.0.jar" />
 				<pathelement location="${dir.lib}/guava-19.0.jar" />
+                <pathelement location="${dir.mod}/modAionImpl.jar" />
```

Comment:
which class needs the modAionImpl as a dependency module?  as the kernel design, modMcf should not involve the modAion/modAionImpl. If you need some class inside the modAionImpl during the implementation, meaning we need to do some refactoring.

Reply:
It seems nothing in modMcf actually uses that right now.  I think I originally added it because I wanted to use CfgConsensusPoW (defined in modAionImpl) from some other Cfg class (defined in modMcf), but I guess I refactored at some point and didn't remove the dependency.  I'll update the PR to take it out.